tests/Frame_Tests.cpp

```diff
@@ -147,4 +147,28 @@ TEST(Copy_Constructor)
 	CHECK_EQUAL(f1.GetAudioSamplesCount(), f2.GetAudioSamplesCount());
 }
 
+#ifdef USE_OPENCV
+TEST(Convert_Image)
+{
+	// Create a video clip
+	std::stringstream path;
+	path << TEST_MEDIA_PATH << "sintel_trailer-720p.mp4";
+	Clip c1(path.str());
+	c1.Open();
+
+	// Get first frame
+	std::shared_ptr<Frame> f1 = c1.GetFrame(1);
```

Comment:
```suggestion
	auto f1 = c1.GetFrame(1);
```
Personally I'm a big proponent of letting the compiler "type" all that crap. :wink:

Reply:
:)